.github/workflows/validate.yml

```diff
@@ -62,6 +62,12 @@ jobs:
 
       - uses: actions/checkout@v4
 
+      - uses: haskell-actions/setup@v2
+        id: setup-haskell
+        with:
+          ghc-version: ${{ matrix.ghc }}
+          cabal-version: '3.10.1.0'
```

Comment:
Should be latest (3.10.2.1?)

Reply:
Probably doesn't matter much, but yes, I guess?

Reply:
I put back `latest` as it was at some point.

Reply:
Next time we change it from `latest`, let's write in a comment (not, e.g., commit or PR description) why we do. :)